### PR TITLE
Bug 1169540 - [Stingray][Shared] Clock should refresh again before se…

### PR DESCRIPTION
**Pull request by sean2449**
…tInterval

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=574d405813664220424be3e7b75bb846e0076743

File: shared/js/smart-screen/clock.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ function Clock() {
       this.timeoutID = window.setTimeout(function cl_setClockInterval() {
 
         if (self.timerID == null) {
+          refresh(new Date());
```
Comment (john-hu) [inline shared/js/smart-screen/clock.js]: I believe this patch works. How about make a patch the same as [1] ? Phone already had it.
[1] https://github.com/mozilla-b2g/gaia/commit/aaf7934bb4bd27f19f1d12434636542a2ab136d4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7dc477f52884c9b89318a63d850512dedd0a08f3

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=40350cf2235b5552cc150622e1bfe0b7aad7419c

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#rMBqrhBwT_K0K4qRpnUfqw

The pull request failed to pass integration tests. It could not be landed, please try again.

**Pull request merged**